packages/site-demo/src/components/layout/Main.tsx

```diff
@@ -1,13 +1,23 @@
+import React from 'react';
+import { Match } from '@reach/router';
 import { ErrorBoundary } from '@lumx/demo/components/ErrorBoundary';
 import { LumxVersion } from '@lumx/demo/components/LumxVersion';
-import { Match } from '@reach/router';
-import React from 'react';
+import { MaterialThemeSwitcher } from '@lumx/react/utils/MaterialThemeSwitcher';
 
 export const Main: React.FC = ({ children }) => (
     <main className="main">
         <div className="main__wrapper">
             <div className="main-header">
-                <Match path="/product/components/*">{({ match }) => match && <LumxVersion />}</Match>
+                <Match path="/product/components/*">
```

Comment:
Why not target everything on "product" as the theme has implications on foundations (fonts), also patterns which use components, and tokens / utilities that may vary?